### DISCUSSION TITLE
Meteor M2 Demodulator

### DISCUSSION THREAD

**Discussion by MM0EFJ** [AlexandreRouma/SDRPlusPlus#726] [General]
Hi,
I have SDR++ v1.0.5 installed on my Raspberry Pi but I cannot see the above plugin, has it been removed form this version?
Regards,
Mike

**Comment by the-gobleo-tinkerer** [comment]
The Meteor Demod is not loaded by default, according to [this table in the main README](https://github.com/AlexandreRouma/SDRPlusPlus#decoders).

- Go to the **Module Manager** tab;
- From the dropdown menu at the bottom, select **meteor_demodulator**;
- In the textbox next to it, put a name for the new tab that will be created for the demod;
- Click on the **+** button.

**Comment by MM0EFJ** [comment]
Hi Shuyuan Liu,
Thanks for the prompt reply, I did as you said but in the Module Manager I
do not have Meteor_Demodulator. They seem to be in alphabetical order and
mine goes from Hack RF Source to Network Sink so no Meteor.
Would I be able to add it ?
Regards,
Mike

**Reply by the-gobleo-tinkerer** [reply]
If you initially installed with a binary package, try updating to the latest successful build from https://github.com/AlexandreRouma/SDRPlusPlus/actions/workflows/build_all.yml: Go to the most recent build with a **√**, scroll down to **Artifacts**, and use the file **sdrpp_raspios_bullseye_armhf**. It should have the Meteor plugin. Make sure to unzip the downloaded file first.

If you built SDR++ from source, repeat the build process, and append `-DOPT_BUILD_METEOR_DEMODULATOR=ON` to the cmake command. Then `make` and `sudo make install`.

It's a bit odd your list starts from "H", there's supposed to be "audio_sink" "airspy_source" "file_source" etc at the front.

**Reply by AlexandreRouma** [reply]
Updating won't add an instance because there is no instance of it by default.
@MM0EFJ I suspect you're confusing the module list with the menu to add a module instance? Go at the bottom of the module manager menu, select "meteor_demodulator" in the drop down, type in a name for the module instance in the text field (eg. "Meteor Demodulator") and click the "+" button. You'll see a new menu appear likely at the bottom of the menu, but you can drag it up to where you want it. When you're done  using it, instead of deleting it you can uncheck the checkbox next to its name which will disable it.

**Reply by the-gobleo-tinkerer** [reply]
> Updating won't add an instance because there is no instance of it by default.

True. I was thinking their installation could be missing the module for some reason, since they said they couldn't find / add it.

@MM0EFJ sorry I should have been clearer. When I said "dropdown" I meant the dropdown next to the "+" button _within_ the Module Manager, not the one that opens Module Manager.
